source/robot_model/include/robot_model/Model.hpp

```diff
@@ -91,56 +90,60 @@ class Model {
    */
   void set_urdf_path(const std::string& urdf_path);
 
-  /**
-   * @brief Getter of the joint names attribute
-   * @return the vector of joint names
-   */
-  const std::vector<std::string>& get_joint_names() const;
-
   /**
    * @brief Getter of the number of joints
    * @return the number of joints
    */
-  unsigned int get_nb_joints() const;
+  const int& get_nb_joints() const;
```

Comment:
there is no point of having a reference for a returned int value. Also, the number of joints can't be negative hence the unsigned int. Maybe you prefer size_t from std?

Reply:
To be honest, I mostly put what CLion suggests me to put. I'll need to learn a bit better what the effects of such statements are. I will check before merging